### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by buningi** [PetrIvan/chord-seq-ai-app#36] [General]
What exactly is this supposed to be useful for? Because as it is. One can't do anything with it. I mean go on youtube and click on any pop or electronic song created in last few years and tell me how this possibly could be useful for doing such. I don't see how to make 1/32 or 1/64 note sounds at all. Maybe I missed something.
If that is available, then I in the wrong. Please show me how that's done.

**Comment by PetrIvan** [comment]
Hi buningi,

Thank you for your feedback and for reaching out with your questions!

ChordSeqAI is designed primarily as a chord progression builder with a focus on harmony, rather than rhythm. The core idea is to help users with harmony-first composition, which has become increasingly popular in electronic and pop music. Essentially you start here with the harmony, export it into MIDI, and load it into a Digital Audio Workstation (DAW) of your choice. In the DAW, you can fully develop your track by adding melody, rhythm, voicing, and other elements that complete the music.

Hence ChordSeqAI addresses just one issue of music composition - how to create a pleasing harmony with a vast space of possible chords. Although it may seem trivial at first, simple rule-based approaches don't work very well if you want anything interesting, so artists often get lost in complex music theory (just look at Jazz harmony, you can literally get a degree in it and still have much to explore). In my experience, once you have this fundamental building block established, it becomes easier to create the melody and other aspects to get your song finished.

Because ChordSeqAI emphasizes harmony, it does not support extremely short note values like 1/32 or 1/64 notes. The note duration is intentionally limited (although there is a [feature request](https://github.com/PetrIvan/chord-seq-ai-app/discussions/22) you can vote for), as it’s intended for users to focus on creating and refining chord progressions rather than detailed rhythmic elements. This design choice allows users to concentrate on the harmonic aspect of their music. In the meantime, you can kind of bypass it by changing the denominator of the time signature to a higher value (such as 4/32) and then decrease it back after resizing the chords (e.g., to 4/4).

If you need any more guidance on using ChordSeqAI or how to integrate it into your workflow, feel free to reach out to us, I'm happy to help!
